packages/common/src/database/LionwebTask.ts

```diff
@@ -0,0 +1,60 @@
+import pgPromise from "pg-promise"
+import { traceLogger } from "../apiutil/index.js";
+import { addRepositorySchema, RepositoryData } from "./DbConnection.js";
+
+/**
+ * All database transactions will go through an instance of this class.
+ * This enables logging, but also tweaking queries when needed.
+ * Current tweak: add the repository schema for each query 
+ * 
+ * This is a wrapper for a pg-promise task.
+ * @see pgPromise.ITask
+ */
+export class LionwebTask {
```

Comment:
Spelling thing: why not `LionWebTask` (capital 'W')?

Reply:
Changed this to LionWebTask